x/evm/keeper/utils.go

```diff
@@ -14,20 +15,26 @@ import (
 	ethtypes "github.com/ethereum/go-ethereum/core/types"
 )
 
+// DefaultPriorityReduction is the default amount of price values required for 1 unit of priority.
+// Because priority is `int64` while price is `big.Int`, it's necessary to scale down the range to keep it more pratical.
+// The default value is the same as the `sdk.DefaultPowerReduction`.
+var DefaultPriorityReduction = sdk.DefaultPowerReduction
```

Comment:
still not sure why this needs to be a separate variable

Reply:
Do you mean to use `sdk.DefaultPowerReduction` directly? I think they serve a different purpose, the user may want to scale to different degree?